qiita_ware/ebi.py

```diff
@@ -6,6 +6,7 @@
 # The full license is in the file LICENSE, distributed with this software.
 # -----------------------------------------------------------------------------
 
+import hashlib
```

Comment:
Minor point but I would think 'from hashlib import md5' is cleaner, since we're only using the one function. Not sure if we have a preference in the lab.

Reply:
In the past, I would always do your recommendation but for whatever reason I decided to go this way. Now, reading online, it seems like there is [no actual benefit (except writing less)](https://stackoverflow.com/a/710603) to go with suggestion. Anyway, I rather leave as it is but I can change if you feel strongly about it; just let me know.